README.md

```diff
@@ -104,5 +104,41 @@ To install a C++ compiler on Windows 10/11, follow these steps:
 3. Download the MinGW installer from the [MinGW website](https://sourceforge.net/projects/mingw/).
 4. Run the installer and select the `gcc` component.
 
+## Docker Version
+
+1. Put your data in models / source_documents in the project root folder (Can be customized changing the corresponding value in the docker-compose.yaml)
+
+2. You can ingest your data using the docker-compose
+
+   1. Without Cuda
+
+    ```sh
+    docker compose run --rm --build privategpt-ingest
+    ```
+
+   2. With Cuda 11.6 or 11.7
+
+    ```sh
+    docker compose run --rm --build privategpt-cuda-11.6-ingest
+
+    docker compose run --rm --build privategpt-cuda-11.7-ingest
```

Comment:
I get an error when running this command, not sure if its related to docker compose version?
```
PS C:\Users\bme\projects\privateGPT> docker compose run --rm --build privategpt-cuda-11.7-ingest
unknown flag: --build
```

Reply:
I am on windows and the flag is ok with (tested from a powershell prompt)
```
docker --version
Docker version 23.0.5, build bc4487a
```

Reply:
Thanks
```
> docker --version
Docker version 20.10.14, build a224086
> docker compose version
Docker Compose version v2.5.1
```
after doing a build, it runs and give this error:
```
> docker compose run --rm privategpt-cuda-11.7-ingest
Traceback (most recent call last):
  File "/home/privategpt/src/ingest.py", line 4, in <module>
    from dotenv import load_dotenv
ModuleNotFoundError: No module named 'dotenv'
```
I see `python-dotenv==1.0.0` in the requirements.txt and the pip install succeeded in the docker build (presumably, cause the build completed and ran).

Reply:
This branch needs a merge from main.

You can do this locally for now:

```
git remote add upstream https://github.com/imartinez/privateGPT
git fetch upstream
git checkout -b local-merge
git merge upstream/main
git add README.md
git commit -m "Ignore README conflicts"
```

Reply:
rebuilt and its working

Reply:
I also received this error, I think it's a red herring due to the output of `pip` being hidden and piped into a log file. This prevents it from erroring when it can't find a package etc. and falls through to trying to run the script.

Solution to prevent this would be to remove the pipe into the log file within `Dockerfile`.